### PR TITLE
Add check for loading assets via openQA-test package

### PR DESCRIPTION
**Pull request by Martchus**
See https://progress.opensuse.org/issues/159411

### CODE REVIEW

File: dist/rpm/openQA-test.spec [hunk 1]
```diff
@@ -23,6 +23,11 @@ touch %{_sourcedir}/%{short_name}
 # call one of the components but not openqa itself which would need a valid
 # configuration
 /usr/share/openqa/script/initdb --help
+
+# verify whether assets can be loaded
+perl -I/usr/share/openqa/lib -mMojolicious -mMojo::Home -mOpenQA::Setup \
+    -e 'OpenQA::Setup::setup_asset_pack(Mojolicious->new(home => Mojo::Home->new("/usr/share/openqa")))'
```
Comment (okurz) [inline dist/rpm/openQA-test.spec:29]:
```suggestion
perl -I/usr/share/openqa/lib -mMojolicious -mMojo::Home -mOpenQA::Setup \
    -e 'OpenQA::Setup::setup_asset_pack(Mojolicious->new(home => Mojo::Home->new("/usr/share/openqa")))' || \
     { echo "assets could not be loaded"; exit $ERRCODE; }
```

Comment (Martchus) [reply dist/rpm/openQA-test.spec:29]: If this command fails it will already print a useful error message. So I wouldn't add another one.

Comment (okurz) [reply dist/rpm/openQA-test.spec:29]: sure but I was trying to get rid of the comment with executable code :)

**Review by okurz — APPROVED** [review]

**Comment by Martchus** [comment]
I now additionally added another commit to make this generally more reliable and less manual.

I created https://github.com/os-autoinst/openQA/pull/5596 to see whether this check would now actually fail.

**Review by baierjan — APPROVED** [review]

**Pull request merged**